contracts/strategies/AaveV3Strategy.sol

```diff
@@ -227,6 +235,61 @@ contract AaveV3Strategy is IController {
      * Strategy methods
      */
 
+    /**
+     * @notice Claims all reward token, then compounds it automatically.
+     * @dev Ops address should send transactions as private in the way prevent exploit from attackers.
+     */
+    function compound() external onlyOps {
+        uint256 _claimable = getUnclaimedReward();
+        if (_claimable == 0) revert NoRewardClaimable();
+
+        aaveReward.claimRewards(supplyingAssets, _claimable, address(this), address(aaveRewardToken));
+
+        uint256 _swapped = _swap(address(aaveRewardToken), address(usdc), aaveRewardToken.balanceOf(address(this)));
+
+        // compound swapped usdc
+        _utilize(_swapped);
+
+        managingFund += _swapped;
+    }
+
+    /**
+     * @notice Gets amount of unclaimed reward token from Aave.
+     */
+    function getUnclaimedReward() public view returns (uint256) {
+        return aaveReward.getUserRewards(supplyingAssets, address(this), address(aaveRewardToken));
+    }
+
+    /**
+     * @inheritdoc OpsReady
+     * @notice Check the rewards can be compounded. If the contract has sufficient reward,
+     *         returns compound() function payload to execute.
+     */
+    function check() external override returns (bool _canExec, bytes memory _execPayload) {
+        uint256 _reward = getUnclaimedReward();
+        uint256 _estimatedOutUsdc = _reward != 0
+            ? exchangeLogic.estimateAmountOut(address(aaveRewardToken), address(usdc), _reward)
+            : 0;
+
+        _canExec = _estimatedOutUsdc >= minOpsTrigger;
+
+        if (_canExec) {
+            _execPayload = abi.encodeWithSelector(this.compound.selector);
```

Comment:
`_estimatedOutUsdc` should be passed to the `compound()` as part of the calldata for slippage control

Reply:
Absolutely. I pushed the new commit.